### DISCUSSION TITLE
Motherboard that runs Windows 11 21H2 but not 22H2

### DISCUSSION THREAD

**Discussion by anzilla** [AveYo/MediaCreationTool.bat#227] [General]
Hello AveYo!

First of all, thank you for the great job that you've done with the tool. It's been very useful to me personally and thousands of other people, I'm sure. 

Please, maybe you will have a tip on the situation that I am facing with one of my motherboards. 

I have two older LGA 1156 motherboards: Asus P7H57D-V EVO and Gigabyte GA-H55M-S2H. Both run on i7 870. 

I installed Windows 11 21H2 using your tool on both of them. Then I updated the Asus one with Windows 11 22H2, simply deleting compatappraiserresources.dll from the sources folder in the installation media. It worked perfectly on this particular moterboard. Windows 11 22H2 runs alright. 

But I have a problem updating the system on the Gigabyte motherboard. First of all, it won't let me update at all if internet is connected. I can go through the process of updating if I unplug the ethernet cable. The installation process finishes and I am even able to start up the system. But as soon as I plug the ethernet back the system restarts and won't work. It loads to repair tools screen every time.

Would you happen to have an idea what may be going on here? Please, point me in the right direction. It is possible that it needs some simple solution. 

Thank you in advance!

Kind regards,

Andzor.

**Comment by anzbanz** [comment]
Hi, this is me again answering my own question. I managed to solve it. The problem was the incompatibility of Windows 11 22H2 with the old ethernet drivers on my motherboard. 

To solve this problem, it is necessary to create a Windows 11 22H2 using Rufus tool. The tool allows bypassing the TPM chip, CPU and RAM checks, as well as bypassing the need to be connected to internet to go through the initial Windows setup. This step is crucial as the unsupported ethernet adapter on the motherboard would not let the process go through. Then, using this installation flash card it is possible both: upgrading from 21H1 or doing a fresh install. But ethernet cable has to be disconnected from the PC, otherwise it will cause a failure to install or restarts of the system and bluescreen. Once the system is installed, I looked up the name of the ethernet chip on my motherboard online, it was a Realtech chip. Then I found some Windows 7 64bit drivers for Realtech ethernet chips and installed them. After this the PC could be connected to Internet with a cable and the OS behaved correctly.

I came across the suggestion to this problem on youtube. Someone explained the main idea that the old ethernet chip on his notebook was casing the same problems. 

I hope this helps someone.

This is me with my different account. For some reason I could not log in to github with my other account name. But the initial post and this update are both done by me. 

Kind regards to everyone,

Andzor.
